### PR TITLE
2 broken links; commas, and other typos.

### PR DESCRIPTION
**Pull request by e-io**
Sprints @EuroPython 2024

### CODE REVIEW

**Comment by e-io** [comment]
Also I've checked now falcon.readthedocs.io for broken links. Overall it's quite good. Two broken links were found only. 

On the page https://falcon.readthedocs.io/en/stable/_modules/index.html :
 - the first link [_asyncio](https://falcon.readthedocs.io/en/stable/_modules/_asyncio.html) does not work. 
 
On the page https://falcon.readthedocs.io/en/stable/user/tutorial-asgi.html :
 - "Hosting our app" -> [Hypercorn](https://pgjones.gitlab.io/hypercorn/)
 
 I may remove them as well if needed. 
 
Screenshot 2024-07-14 at 18 32 49

 The check was performed with this wonderful tool:
 https://github.com/stevenvachon/broken-link-checker (`blc -ro <a-website-with-links>`)

**Comment by e-io** [comment]
With a tool mentioned above I have carefully tidied up **all** the [Falcon Wiki](https://github.com/falconry/falcon/wiki/) as well. However, most pages had just 2-5 links to fix. 

Screenshot 2024-07-14 at 20 08 09

**Comment by CaselIT** [comment]
that tool seems very useful! tanks for linking it.

I'll try looking at this and the other links asap (I'm very busy this week sadly)

thank you for the effort!

**Comment by vytas7** [comment]
Thanks for the effort, I'll try to fix the links in the main repo. IIRC Hypercorn was moved from Gitlab to GitHub, hence the issue. 
Regarding Wiki, ideally we shouldn't try just removing them, but finding if they have been relocated or pasted wrong. For instance, the Documatt blog link is still there, maybe we just need to get a different permalink.

**Review by vytas7 — APPROVED** [review]
Looks great, thanks a lot! :100: I really like the attention to the details.
We are planning on rebuilding the website using some kind of generator like Sphinx, MkDocs, etc in order to reduce the maintenance burden, but we'll probably still keep a fair bit of content, so your work wouldn't be lost.

File: src/index.html [hunk 1]
```diff
@@ -99,7 +93,7 @@ <h1 class="block-title display-1">Falcon</h1>
         <p>
             <a href="https://www.kontrolnaya-rabota.ru/s/" target="_blank"><img src="img/sponsors/rabota.jpg" height="30" alt="Examination RU" style="margin-right: 10px"></a>
 
-            <a href="https://www.pnk.sh/python-falcon" target="_blank"><img src="img/sponsors/paris.svg" height="30" alt="Paris Kejser" style="margin-right: 10px"></a>
+            <a href="www.youtube.com/c/ParisNakitaKejser" target="_blank"><img src="img/sponsors/paris.svg" height="30" alt="Paris Kejser" style="margin-right: 10px"></a>
```
Comment (vytas7) [inline src/index.html:96]: Ah, we missed the scheme there. I'll fix it.

**Comment by e-io** [comment]
@vytas7 

> "Regarding Wiki, ideally we shouldn't try just removing them, but finding if they have been relocated or pasted wrong."


As for links in "Who's using Falcon?": 

Initially I was trying to find new links of these projects, but, then, I found that most of them don't exist anymore. Even if any project might exist with a new web address, there is no guarantee that it is still based on Falcon. 

Anyway, I hope that the remaining list "Who's using Falcon?" is still enough huge :)

**Pull request merged**